src/index.html

```diff
@@ -252,9 +268,27 @@
     bearing: config.chapters[0].location.bearing,
     pitch: config.chapters[0].location.pitch,
     interactive: false,
-    transformRequest: transformRequest
+    transformRequest: transformRequest,
+    projection: config.projection
 });
 
+// Create a inset map if enabled in config.js
+if (config.inset) {
+ var insetMap = new mapboxgl.Map({
+    container: 'mapInset', // container id
+    style: 'mapbox://styles/mapbox/dark-v10', //hosted style id
+    center: config.chapters[0].location.center,
+    // Hardcode above center value if you want insetMap to be static.
+    zoom: 3, // starting zoom
+    hash: false,
+    interactive: false,
+    attributionControl: false,
+    //Future: Once official mapbox-gl-js has globe view enabled,
+    //insetmap can be a globe with the following parameter.
+    //projection: 'globe'
```

Comment:
https://user-images.githubusercontent.com/12744420/155816470-d03d6a67-8783-4270-9ef5-31f759ad82e5.mov

While the globe view right now is not available in the official release, I tested it out with a local build and it looks great.